src/material/tabs/tab-group.ts

```diff
@@ -242,6 +242,12 @@ export class MatTabGroup implements AfterContentInit, AfterContentChecked, OnDes
 
   private _backgroundColor: ThemePalette;
 
+  /** Aria label of the inner `tablist` of the group. */
+  @Input('aria-label') ariaLabel: string;
```

Comment:
I'm a bit on the fence if this should be `aria-label` or something like `tabListAriaLabel`. I went with the shorter one, because the `mat-tab-group` doesn't have a `role`.

Reply:
It also seems some of out other components (like [autocomplete](https://material.angular.io/components/autocomplete/api#MatAutocomplete)) follow the same too with just calling the input ariaLabel